app/assets/src/components/views/compare/SamplesHeatmapView.jsx

```diff
@@ -690,8 +690,13 @@ class SamplesHeatmapView extends React.Component {
   handleDownloadClick = fileType => {
     switch (fileType) {
       case "svg":
+        // TODO (gdingle): pass in filename per sample?
         this.heatmapVis.download();
         break;
+      case "png":
+        // TODO (gdingle): pass in filename per sample?
```

Comment:
What do you mean by these?

Reply:
Make the filename unique for the set of sample selected.

Reply:
I would not add comments to the code for things that we *might* consider doing... a JIRA task might be better for that... 
If you still keep the comment, at least change to "per sample set".

Reply:
Yes, I put some of these comments in for discussion in review. I'll make this one a low pri JIRA task.